src/gc/gc.d

```diff
@@ -498,15 +563,12 @@ struct GC
 
 
     //
-    //
+    // Implementation for malloc and calloc.
     //
     private void *mallocNoSync(size_t size, uint bits, ref size_t alloc_size, const TypeInfo ti = null) nothrow
     {
         assert(size != 0);
-
-        //debug(PRINTF) printf("GC::malloc(size = %d, gcx = %p)\n", size, gcx);
```

Comment:
I'd recommend keeping the commented logging of modifying API calls, they can be helpful when trying to find bugs. Some need to be corrected, though.